### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by muskan2532150**
- create animals table.
 - Insert Values 
 - Queries Result

- Query 1

Query 1
- Query 2

 
- Query 3

- Query 4

 - Query 5
 

- Query 6

-  Query 7

-  Query 8

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals where name like '%mon' ;
+SELECT name from animals where EXTRACT(YEAR FROM date_of_birth) between 2016 AND 2019;
+SELECT name from animals where neutered='true' AND escape_attempts>='3';
```
Comment (fobadara) [inline queries.sql:5]: [Required] You were expected to get animals that are neutered and escape attempts is less than 3 not more than or equal to

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals where name like '%mon' ;
+SELECT name from animals where EXTRACT(YEAR FROM date_of_birth) between 2016 AND 2019;
+SELECT name from animals where neutered='true' AND escape_attempts>='3'; 
+SELECT date_of_birth from animals where name='Agumon' OR name='pikachu';
+SELECT name,escape_attempts from animals where weight_kg>='10.5';
```
Comment (fobadara) [inline queries.sql:7]: [Required] Where the. weight is more than 10.5kg not more than or equal to.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @muskan2532150, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good first attempt

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @muskan2532150,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Highlights
- Descriptive pull request message ✔️
- Professional readme file ✔️


image2
# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @muskan2532150 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**